src/libraries/System.Private.CoreLib/src/System/Security/IStackWalk.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace System.Security
 {
+#if NETCOREAPP // This file is compiled into both System.Private.CoreLib (NetCoreApp) and System.Security.Permission (netstandard2.0)
```

Comment:
```suggestion
#if SYSTEM_PRIVATE_CORELIB
```
It will make the code self-describing. We are using this ifdef in number of other similar situations in CoreLib.